stemcell_builder/stages/bosh_go_agent/apply.sh

```diff
@@ -6,9 +6,14 @@ source $base_dir/lib/prelude_apply.bash
 source $base_dir/lib/prelude_bosh.bash
 
 mkdir -p $chroot/etc/sv
+mkdir -p $chroot/var/vcap/monit/svlog
+
+cp -a $assets_dir/nats-access-helper.sh $chroot/$bosh_dir/etc/nats-access-helper.sh
+cp -a $assets_dir/restrict-nats-api-access-allow-monit-api $chroot/etc/network/if-up.d/restrict-nats-api-access-allow-monit-api
 cp -a $assets_dir/runit/agent $chroot/etc/sv/agent
 cp -a $assets_dir/runit/monit $chroot/etc/sv/monit
-mkdir -p $chroot/var/vcap/monit/svlog
+
+chmod +x $chroot/$bosh_dir/etc/nats-access-helper.sh $chroot/etc/network/if-up.d/restrict-nats-api-access-allow-monit-api
```

Comment:
as we are planing to move to netplan in JJ and ifupdown is specificly used here.
do we have a easy way to solve this?

Reply:
looks like a similar hook is available according to:
https://netplan.io/faq/

Reply:
@ramonskie what is the right branch for this PR? I based of bionic/master, should I recreate the PR based on ubuntu-bionic/1.x?

Reply:
the ubuntu-bionic/master is fine as we then port it/rebase it to 1.x